docs/markdown/Getting Help/the-pants-community/team.md

```diff
@@ -6,7 +6,11 @@ hidden: false
 createdAt: "2022-08-15T22:36:48.659Z"
 updatedAt: "2022-08-28T22:13:54.279Z"
 ---
-Pants open source project has had many team members in over a decade. The current team is comprised of elected Contributors and Maintainers, who are nominated based on [criteria](docs:the-pants-community#how-is-the-community-structured) including demonstrated commitment to the project and a track record of contributions to the project and community. Interested in joining our team? Learn more on [The Pants Community](docs:the-pants-community) page.
+> 📘 Interested in joining Pants team?
```

Comment:
This should probably go below the paragraph on line 13. First define the team, then offer that you can join it?

Reply:
That was my initial approach. But since the goal is to make the team feel welcoming right off the bat, I didn't like having the invitations land so far down the page. It gave the impression of having been buried, which runs counter to the intended takeaway. Does it seem out-of-place now? I didn't think so, but if it does I can go back and work on rejiggering the page to bring the definitions higher.

Reply:
Yeah, I think it seems out of place. Only one paragraph lower down doesn't seem very buried?